openedx/features/teams/views.py

```diff
@@ -176,7 +176,7 @@ def view_team(request, course_id, team_id):
     room_id = team_group_chat.room_id
     leave_team_url = reverse('team_membership_detail', args=[team_id, user.username])
 
-    team_administrator = (has_access(user, 'staff', course.id)
+    team_administrator = (has_access(user, 'staff', course.id, None, False)
```

Comment:
can we use named parameter here and in below changes to avoid passing None

Reply:
done

Reply:
it still exists

Reply:
done